src/responsiveHOC.js

```diff
@@ -28,7 +28,7 @@ function responsiveHOC (wait = 150, debounceOptions) {
       }
 
       render () {
-        return <Component {...this.props} {...this.state} />
+        return <Component ref='lineEllipsisChild' {...this.props} {...this.state} />
```

Comment:
[String refs API is legacy.](https://reactjs.org/docs/refs-and-the-dom.html#legacy-api-string-refs)